src/lib/temple/front/use-quipu-tokenlist.hook.ts

```diff
@@ -0,0 +1,28 @@
+import { useCallback, useEffect } from 'react';
+
+import { useRetryableSWR } from 'lib/swr';
+import { usePassiveStorage } from 'lib/temple/front';
+import { getQuipuWhitelist, TokenListItem } from 'lib/templewallet-api/token-list';
+
+export const useQuipuTokenlist = () => {
```

Comment:
you are adding these tokens on Swap & Manage Screens separately.  
Maybe add these tokens here https://github.com/madfish-solutions/templewallet-extension/blob/development/src/lib/temple/front/sync-tokens.ts#L125
After this sync, they will be available everywhere.